File: config/config-sbc2ha-new/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/platform/Ds2482BusConfig.java
package com.dfi.sbc2ha.config.sbc2ha.definition.platform;

import com.dfi.sbc2ha.config.sbc2ha.definition.enums.PlatformType;
import lombok.Data;

@Data
public class Ds2482BusConfig extends I2cBusConfig {
    public Ds2482BusConfig() {
        super();
        platform = PlatformType.DS2482;
    }
}

File: config/editor/src/app/definition/platform/bus/Ds2482BusConfig.ts
import {I2cBusConfig} from "./I2cBusConfig";
import {PlatformType} from "../../enums/PlatformType";


export class Ds2482BusConfig extends I2cBusConfig {
    public constructor(busId: string, address: number) {
        super(PlatformType.DS2482, busId, address);
    }
}
